src/main/resources/templates/mdBenefitsFlow/ohepThankYou.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="~{fragments/head :: head(title=#{ohep-thank-you.title})}"></head>
+<body>
+<div class="page-wrapper">
+  <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+  <section class="slab">
+    <div class="grid">
+      <div th:replace="~{fragments/goBack :: goBackLink}"></div>
+      <main id="content" role="main" class="form-card spacing-above-35">
+        <div class="center text--align-center">
+          <th:block
+              th:replace="~{fragments/cardHeader :: cardHeader(header=#{ohep-thank-you.header})}"/>
+          <img src="/assets/images/ohep-logo.png" alt="Logo for the Office of Home Energy Programs" class="ohep-logo" style="height: 33%; width: 33%;">
```

Comment:
Noting that Devon changed this to "Logo for the Office of Home Energy Programs, Your Home Energy Partner".  Will it work to put that into the messages.properties file?